db/Pos/DB/Class.hs

```diff
@@ -91,6 +92,14 @@ class DBIteratorClass i where
 
 type IterType i = (IterKey i, IterValue i)
 
+newtype RawBlock = RawBlock
+    { unRawBlock :: ByteString
+    }
+
+newtype RawUndo = RawUndo
+    { unRawUndo :: ByteString
```

Comment:
Imo, it'd be better to have `newtype Serialized a = Serialized ByteString`, but for CSL-1854 this solution is enough.

Reply:
I don't think so, I introduced these wrappers to avoid mismatching `Block` and `Undo` ByteStrings

Reply:
Pleeease can you _at least_ rename them? We're discussing this CSL-1859 issue and we operate with term `RawBlock` meaning that is an "unverified block", not a "serialized" one. 
I actually find Ivan's idea to be a pretty good thing, because CSL-1869 will anyway introduce a notion of `Serialized a` as Ivan has proposed -- so if you do it now, you minimize the diff.

Reply:
I propose `newtype Serialized a = Serialized ByteString`

Reply:
lol, @gromakovsky it seems you edited your first comment

Reply:
The first comment by gromak was there in the same form yesterday evening.
`Serialized a` works for me.

Reply:
> it seems you've edited your first comment

No, if I edited it, github would show it.